pkg/cmd/exec/exec.go

```diff
@@ -0,0 +1,26 @@
+package exec
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+// NewExecShim returns a cobra command that shells out the specified target with exactly the same args
+func NewExecShim(name, target string, knownArgs ...string) *cobra.Command {
+	return &cobra.Command{
+		Use:                name,
+		DisableFlagParsing: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			execCmd := exec.Command(target, append(knownArgs, args...)...)
```

Comment:
You need to execve, not exec directly.

Reply:
https://github.com/openshift/origin/blob/master/pkg/cmd/server/start/start_node.go#L394